modules/core/packet-server/keeper/keeper.go

```diff
@@ -19,3 +27,76 @@ func NewKeeper(cdc codec.BinaryCodec, channelKeeper types.ChannelKeeper, clientK
 		ClientKeeper:  clientKeeper,
 	}
 }
+
+func (k Keeper) SendPacket(
+	ctx sdk.Context,
+	_ *capabilitytypes.Capability,
+	sourceChannel string,
+	sourcePort string,
+	destPort string,
```

Comment:
destPort must be passed in but destChannel does not since we have it in the counterparty